src/views/List.jsx

```diff
@@ -1,15 +1,45 @@
 import { ListItem } from '../components';
+import { useState } from 'react';
 
 export function List({ data }) {
+	const [searchItem, setSearchItem] = useState('');
+
+	const handleTextChange = (event) => {
+		setSearchItem(event.target.value);
+	};
+
+	const filteredItems = data.filter((item) => {
+		return item.name.toLowerCase().includes(searchItem.toLowerCase());
+	});
+
 	return (
 		<>
-			<p>
-				Hello from the <code>/list</code> page!
-			</p>
+			<form onSubmit={(event) => event.preventDefault()}>
+				<label htmlFor="search-item">Search Item: </label>
+
+				<input
+					id="search=item"
```

Comment:
It looks like a mismatch between htmlFor property on line 18 and input id on line 21

Reply:
Good catch @kweeuhree! 

I think this should be changed to match the `htmlFor` before this can be merged into `main`.
```suggestion
					id="search-item"
```